web/themes/custom/kitrab/src/scss/_life-news.scss

```
@@ -0,0 +1,11 @@
+.node--view-mode-image-title-date {
+  .image-style-medium-content {
+    min-height: 150px;
+  }
+  .field--name-title {
+    font-size: rem(12);
+    h2 {
+      line-height: rem(18);
+    }
+  }
+}
```

Need a blank space at the end of the file.
```suggestion
}

```